src/main/kotlin/app/hashers/CommitHasher.kt

```diff
@@ -47,20 +56,40 @@ class CommitHasher(private val localRepo: LocalRepo,
     private fun hashAndSendCommits() {
         val lastKnownCommit = repo.commits.lastOrNull()
         val knownCommits = repo.commits.toHashSet()
+
+        val factsDayWeek = hashMapOf<Author, Array<Int>>()
+        val factsDayTime = hashMapOf<Author, Array<Int>>()
```

Comment:
btw, what's difference between hashMapOf and mutableMap?

Reply:
MutableMap is an interface and mutableMapOf create default implentation of map, which is for kotlin 1.1.4-2 LinkedHashMap.
hashMapOf and linkedMapOf clearly create HashMap and LinkedHashMap which implement MutableMap interface.